lisa/tools/qemu_img.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+
+
+class QemuImg(Tool):
+    @property
+    def command(self) -> str:
+        return "qemu-img"
+
+    def create_diff_qcow2(self, output_img_path: str, backing_img_path: str) -> None:
+        params = f"create -F qcow2 -f qcow2 -b {backing_img_path} {output_img_path}"
+        self.run(
+            params,
+            expected_exit_code=0,
```

Comment:
You should probably set `force_run=True` here. (Tools by default will cache and reuse the results of identical commands.)